src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -31,8 +46,8 @@ const CrossPodQueryForm = () => {
   const handleCrossPodQuery = async (event) => {
     event.preventDefault();
     dispatch({ type: 'SET_PROCESSING' });
-    const docType = event.target.document.value;
-    let podUsername = event.target.crossPodQuery.value;
+    const docType = documentType;
+    let podUsername = searchUser;
```

Comment:
Related to Line 32, we already have a state that grabs our username. Line 29 also contains the onChange function within {...username}. Revert this back.